alchemy-web/docs/providers/sentry/client-key.md

```diff
@@ -31,6 +31,65 @@ Get your auth token from [Sentry's API settings](https://sentry.io/settings/acco
 
 ## Usage
 
+```typescript
+import { ClientKey } from "@alchemy/providers/sentry";
+
+// Create a basic client key
+const key = await ClientKey("my-key", {
+  name: "My Key",
+  project: "my-project",
+  organization: "my-org"
+});
+
+// Create a client key with rate limiting
+const key = await ClientKey("rate-limited-key", {
+  name: "Rate Limited Key",
+  project: "my-project",
+  organization: "my-org",
+  rateLimit: {
+    window: 3600, // 1 hour
+    count: 1000   // 1000 events per hour
+  }
+});
+
+// Create a client key with a custom auth token
+const key = await ClientKey("my-key", {
+  name: "My Key",
+  project: "my-project",
+  organization: "my-org",
+  authToken: alchemy.secret("my-auth-token")
+});
+```
```

Comment:
nit: Each snippet should be under a `##` heading so they are included in the TOC

Same convention as:

https://github.com/sam-goodwin/alchemy/blob/main/alchemy-web/docs/providers/stripe/price.md

Reply:
Resolved via 4f8853d082d163233a6800906ce39917e4981653

(Looking forward to autogen)